### PR TITLE
Add LinkStack icon

### LINKED ISSUE #11144: Request: LinkStack
### Brand Name

LinkStack

### Website

https://linkstack.org

### Popularity Metric

https://www.similarweb.com/website/linkstack.org

### Official Resources for Icon and Color

https://raw.githubusercontent.com/LinkStackOrg/branding/main/logo/svg/logo_animated.svg

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by MuhammadZariyanAsif123**
**Issue:** closes #11144.

**Popularity metric:**

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [ ] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by MuhammadZariyanAsif123** [comment]
The "LinkStack" information has been added in _data/simple-icons.json.

**Comment by uncenter** [comment]
Please read our [contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon) about how to add an icon. You need to properly optimize and simplify the icon.

File: _data/simple-icons.json
Comment (uncenter) [inline _data/simple-icons.json:1]: There are extra formatting changes here, Please remove the unnecessary newlines.

File: icons/linkstack.svg
Comment (uncenter) [inline icons/linkstack.svg:1]: See earlier comment.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/linkstack.svg
Comment (MuhammadZariyanAsif123) [reply icons/linkstack.svg:1]: Actually, there is some misunderstanding, these are not extra spaces.
The first screen is displaying the old code and the other one shows the updated code.

Comment (uncenter) [reply icons/linkstack.svg:1]: What? You need to simplify this SVG, combine the paths, remove any fill/stroke colors, and optimize it.

File: _data/simple-icons.json
Comment (uncenter) [reply _data/simple-icons.json:1]: You have not removed the extra formatting. There is are extra newlines on line 8556 and line 8562.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8553,6 +8553,13 @@
             "hex": "B6A272",
             "source": "https://www.lamborghini.com"
         },
+         
+        {
+            "title": "LinkStack",
+            "hex": "B6A272",
+            "source": "https://linkstack.org"
+        },
+
```
Comment (uncenter) [inline _data/simple-icons.json:8562]:
```suggestion
        {
            "title": "LinkStack",
            "hex": "B6A272",
            "source": "https://linkstack.org"
        },
```

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by uncenter** [comment]
Closing as stale, and the brand/project is not within our popularity metrics - see original issue.

**Pull request closed without merging**